src/KnownRelaysManager.ts

```diff
@@ -254,29 +208,23 @@ export class KnownRelaysManager {
 
     async _sortRelaysInternal(
         transactionDetails: EnvelopingTransactionDetails,
-        activeRelays: RelayInfoUrl[]
-    ): Promise<RelayInfoUrl[]> {
+        activeRelays: RelayManagerData[]
+    ): Promise<RelayManagerData[]> {
         const scores = new Map<string, number>();
         for (const activeRelay of activeRelays) {
             let score = 0;
-            if (isInfoFromEvent(activeRelay)) {
-                const eventInfo = activeRelay as RelayRegisteredEventInfo;
-                score = await this.scoreCalculator(
-                    eventInfo,
-                    transactionDetails,
-                    this.relayFailures.get(activeRelay.relayUrl) ?? []
-                );
-                scores.set(eventInfo.relayManager, score);
-            }
+            score = await this.scoreCalculator(
+                activeRelay,
+                transactionDetails,
+                this.relayFailures.get(activeRelay.url) ?? []
+            );
+            scores.set(activeRelay.manager, score);
         }
-        return Array.from(activeRelays.values())
-            .filter(isInfoFromEvent)
```

Comment:
I think this filter removal is related to [thishttps://github.com/infuy/rif-relay-client/pull/3/files/ed0e24817666109cf0031f413d7a4cbac0ae5a22#r742923898)